realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -462,6 +477,15 @@ private static String getManagementRealmUrl(URL authUrl) {
         }
     }
 
+    /**
+     * FIXME Javadoc
+     *
+     * @return
+     */
+    public PermissionManager getPermissionManager() {
```

Comment:
Can I suggest making PermissionsManager an interface, in `io.realm`, and them moving most of this stuff to `io.realm.internal.somewhere`.  `io.realm` is already a swamp...